index.html

```diff
@@ -1,417 +1,244 @@
-                <div class="interest">
-                    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in
-                        touch.
-                    </h2>
-                </div>
-                <div>
-
-                    <form action="https://formspree.io/f/mrgjeazo" class="form" id="form" method="post">
-                        <ul>
-                            <li class="border-bottom">
-                                <input type="text" id="name" placeholder="Full name" required maxlength="30">
-                            </li>
-                            <li class="border-bottom">
-                                <input type="email" name="user-email" id="email" placeholder="Email address" required>
-                            </li>
-                            <li>
-                                <textarea name="comment" id="comment" placeholder="Enter text here" maxlength="500"
-                                    required></textarea>
-                                <small id="error"></small>
-                            </li>
-                            <li class="btn-touch border-btn">
-                                <button class="submit" type="submit">Get in touch</button>
-                            </li>
-                        </ul>
-                    </form>
-                </div>
-            </section>
-        </section>
-        <footer>
-            <hr class="bg-footer">
-            <ul class="footer-social">
-                <li> <a href="https://www.github.com/telanet"><img src="images/Icon-GitHub1.png" alt="github"></a> </li>
-                <li><a href="https://www.linkedin.com/in/benard-ngetich"><img src="images/Icon-Linkedin1.png"
-                            alt="Linkedin">
-                    </a></li>
-                <li><a href="#"> <img src="images/Icon-Angelist1.png" alt="Angelist"></a> </li>
-                <li><a href="https://www.twitter.com/Ben54647605"><img src="images/Icon-Twitter1.png" alt="twitter"> </a>
-                </li>
-                <li><a href="#"><img src="images/Icon-Medium1.png" alt="Medium"> </a> </li>
+          </div>
+        </div>
+      
+      </section>
+
+      <section class="projects">
+        
+
+      </section>
+
+      <section class="page-section aboutme-section">
+        <h2 class="section-title title-l1">About me</h2>
+        <p class="description-text">Hello I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+          to contact me.</p>
+        <button type="button" name="button" class="enable-button basic-button">Get My Resume</button>
+        <hr class="h-rule">
+        <ul class="skills ul-list vertical-list">
+          <li class="list-item-text languages-skills">
+            <span class="sublist-title">Languages</span>
```

Comment:
- [OPTIONAL] As required by the last reviewers. Kindly add the orange symbol to the headings to replicate the Figma design. 

### Your Design

![your-frameworks](https://user-images.githubusercontent.com/26597432/193420322-635ad490-ceb7-47a5-b777-e7710fa7037b.png)

### Figma Design

![figma-frameworks](https://user-images.githubusercontent.com/26597432/193420387-335cd227-fc42-4061-bae1-e3d81585c8a8.png)